DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/static_math.hpp

```diff
@@ -1,5 +1,12 @@
-#ifndef __MISC_HPP__
-#define __MISC_HPP__
+#ifndef __STATICMATH_HPP__
+#define __STATICMATH_HPP__
+
+#define SWAP(a, b)  \

```

Comment:
Does this function really belong in 'static math', since it appears to be neither math nor static?

Reply:
Can I rename the file "common_utilities.hpp"?? Would that address the problem?

Reply:
File renamed to impu_math.hpp